docs/resources/samples/send-teams-invite-from-excel-data.md

```diff
@@ -0,0 +1,237 @@
+---
+title: 'Send a Teams meeting from Excel data'
+description: 'Learn how to use Office Scripts to send a Teams meeting from Excel data.'
+ms.date: 03/30/2021
+localization_priority: Normal
+---
+
+# Send Teams meeting from Excel data
+
+This solution shows how to use Office Scripts and Power Automate actions to select rows from Excel file and use it to send a Teams meeting invite then update Excel.
+
+## Example scenario
+
+* An HR recruiter manages the interview schedule of candidates in an Excel file.
+* The recruiter needs to send the Teams meeting invite to the candidate and interviewers. The business rules are to select:
+
+    (a) Invites to only those for whom the invite isn't already sent as recorded in the file column.
+
+    (b) Interview dates in the future (no past dates).
+
+* The recruiter needs to update the Excel file with the confirmation that all Teams meetings have been sent for the eligible records.
+
+## Solution
+
+The solution has 3 parts:
+
+1. Office Script to extract data from a table based on conditions and returns an array of objects as JSON data.
+1. The data is then sent to the Teams **Create a Teams meeting** action to send invites. Send one Teams meeting per instance in the JSON array.
+1. Send the same JSON data to another Office Script to update the status of the invitation.
+
+## Sample Excel file
+
+Download the file <a href="hr-schedule.xlsx">hr-schedule.xlsx</a> used in this sample and try it out yourself!
+
+## Office Scripts
```

Comment:
See previous comment about unnecessary navigation. It adds a heading level (bad for SEO) and makes it difficult in scan mode for screen readers.